guide/ops/troubleshooting/increase-entropy.md

```diff
@@ -4,24 +4,52 @@ title: Increase Entropy
 toc: /guide/toc.json
 ---
 
-If you are installing Apache Brooklyn on a virtual machine, you may find it useful to increase the 
-Linux kernel entropy to speed up the ssh connections to the managed entities. You can install and 
+### Checking entropy level
+
+A lack of entropy can cause random number generation to be extremely slow.
+This results in tasks like ssh to also be extremely slow.
+Check the available entropy on the Apache Brooklyn machine with.
```

Comment:
Reword to "One can check the available entropy on a machine by running the command:"